website/versioned_docs/version-19.0.0/smart_rollups.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
we probably should remove this, the operation is not available in 19.0.0. It will  probably be available from 19.1.0 onwards